project/resources/tests.py

```diff
@@ -1,88 +1,110 @@
+        create(ResourceFactory, title="elm Documentation")
+        create(ResourceFactory, description="The best place to start is the official guide. It will give you a solid foundation for creating applications with Elm. Once you have worked through that, the next place to look for documentation is on the packages you are using.")
+        create(ResourceFactory, description="Elm is a programming language targeted at the front end (runs in a browser, not on Node) that offers a different take on building dynamic web applications.")
+
         url = '/api/v1/resources/?search=Elm'
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(response.data['count'], 3)
+        self.assertEqual(response.data['count'], 4)
         self.assertContains(response, "Elm Crash-Course (Reactivate London)")
+        self.assertContains(response, "or creating applications with Elm. ")
 
 
     def test_view_one_resource(self):
+        new_resource = create(ResourceFactory, guid='96f1ee80-59f4-11ea-9149-dca9047779fe')
         url = '/api/v1/resources/96f1ee80-59f4-11ea-9149-dca9047779fe/'
+
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(response.data['title'],"Linear Digressions")
-        self.assertEqual(response.data['description'], "Linear Digressions is a podcast about Machine Learning and Data Science")
+        self.assertEqual(response.data['title'], new_resource.title)
+        self.assertEqual(response.data['description'], new_resource.description)
 
 
     def test_patch_one_resource(self):
-        url = '/api/v1/resources/96f5e4ea-59f4-11ea-9149-dca9047779fe/'
+        new_resource = create(ResourceFactory, guid='96f1a740-59f4-11ea-9149-dca9047779fe')
+        url = f'/api/v1/resources/{new_resource.guid}/'
 
         data= {
             "description": "A _diabolically irresponsible_ talk in which I celebrate modern Python coding by **abandoning all backwards compatibility** THE END",
-            "tags": ["test tags", "testing"]
+            "author": "David Beazley"
         }
+
         response = self.client.patch(url, data, format='json')
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(response.data['description'], data['description'])
-        self.assertEqual(response.data['tags'][0]["name"], data['tags'][0])
+        self.assertEqual(response.data['author'], data['author'])
 
 
     def test_delete_one_resource(self):
-        #to do:  grab the guid from the created guids in the DB instead of hard coding it.
-        url = '/api/v1/resources/96eec6e2-59f4-11ea-9149-dca9047779fe/'
+        records = create_batch(ResourceFactory, 10)
+        new_resource = create(ResourceFactory, guid='96f4dd16-59f4-11ea-9149-dca9047779fe')
+        url = f'/api/v1/resources/{new_resource.guid}/'
```

Comment:
Because you're using `new_resource.guid`, we don't actually need to hard-code the guid. This could just be:

```py
new_resource = create(ResourceFactory)
url = f'/api/v1/resources/{new_resource.guid}/'
```

Also, same comment for `test_view_one_resource()`

Reply:
Good point. Dunno what I was thinking there...I did that elsewhere, but brain farted here.  Fixed in the new push.  Thank for catching!